recipes/libmodbus/all/conanfile.py

```diff
@@ -23,101 +31,116 @@ class LibmodbusConan(ConanFile):
         "fPIC": True,
     }
 
-    _autotools = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _settings_build(self):
-        return self.settings_build if hasattr(self, "settings_build") else self.settings
-
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+            try:
+                del self.options.fPIC # once removed by config_options, need try..except for a second del
+            except Exception:
+                pass
+        try:
+            del self.settings.compiler.libcxx # for plain C projects only
+        except Exception:
+            pass
+        try:
+            del self.settings.compiler.cppstd # for plain C projects only
+        except Exception:
+            pass
```

Comment:
```suggestion
            pass
        if self.settings.os == "Windows":
            self.win_bash = True
```
It should solve the question related to unix path.